kml/types.hpp

```diff
@@ -65,6 +68,33 @@ inline std::string DebugPrint(PredefinedColor color)
   UNREACHABLE();
 }
 
+inline dp::Color ColorFromPredefinedColor(PredefinedColor color)
+{
+  switch (color)
+  {
+    case kml::PredefinedColor::Red: return dp::Color(229, 27, 35, 255);
+    case kml::PredefinedColor::Pink: return dp::Color(255, 65, 130, 255);
+    case kml::PredefinedColor::Purple: return dp::Color(155, 36, 178, 255);
+    case kml::PredefinedColor::DeepPurple: return dp::Color(102, 57, 191, 255);
+    case kml::PredefinedColor::Blue: return dp::Color(0, 102, 204, 255);
+    case kml::PredefinedColor::LightBlue: return dp::Color(36, 156, 242, 255);
+    case kml::PredefinedColor::Cyan: return dp::Color(20, 190, 205, 255);
+    case kml::PredefinedColor::Teal: return dp::Color(0, 165, 140, 255);
+    case kml::PredefinedColor::Green: return dp::Color(60, 140, 60, 255);
+    case kml::PredefinedColor::Lime: return dp::Color(147, 191, 57, 255);
+    case kml::PredefinedColor::Yellow: return dp::Color(255, 200, 0, 255);
+    case kml::PredefinedColor::Orange: return dp::Color(255, 150, 0, 255);
+    case kml::PredefinedColor::DeepOrange: return dp::Color(240, 100, 50, 255);
+    case kml::PredefinedColor::Brown: return dp::Color(128, 70, 51, 255);
+    case kml::PredefinedColor::Gray: return dp::Color(115, 115, 115, 255);
+    case kml::PredefinedColor::BlueGray: return dp::Color(89, 115, 128, 255);
+    case kml::PredefinedColor::None:
```

Comment:
This mapping between the predefined and dp::Color was placed in the ios part (I dunno why).
1. I think that it will be better to move it to the cpp (because both types are in the core).
2. For now it is used to get the dp::Color from the predefined color only for the color generator.

Reply:
I agree about moving into cpp. 

The question was: how None color is used in UI? Should it return some transparent color instead of a red one?

Reply:
Or maybe fail immediately?

Reply:
I saw that this approach is already used in the bookmarks:

https://github.com/organicmaps/organicmaps/blob/6b242fcb009be3d24a81e63ae1f0e06b2c0b8044/map/bookmark.cpp#L142-L183

Reply:
My question was "how to handle the None color". Won't make it Red break something?

Reply:
@biodranik I don't know if asserting the None color breaks the logic somewhere or not. It requires some time to investigate all the possible cases because this color is used in multiple places. 
This is why the default (red) RGB color is returned when the predefined color cannot be parsed successfully.